c10/core/StorageImpl.h

```diff
@@ -222,5 +223,6 @@ struct C10_API StorageImpl : public c10::intrusive_ptr_target {
   // local to process cuda memory allocation
   bool received_cuda_;
   Allocator* allocator_;
+  impl::PyObjectSlot pyobj_slot_;
```

Comment:
@swolchok gonna be so mad 🥲 (this adds two words per storage which is not nothing, but I don't really know how else we can do this PR set)

Reply:
are there platforms (like, say, `C10_MOBILE`) where we can statically identify that we don't need this?

Reply:
Any platform that doesn't use Python should not need the PyObjectSlot. I'm not sure if there are any other than `C10_MOBILE`. I can add an ifdef if you'd like